carma_wm/include/carma_wm/CARMAWorldModel.h

```diff
@@ -105,6 +105,10 @@ class CARMAWorldModel : public WorldModel
   /*! \brief Set endpoint of the route
    */
   void setRouteEndPoint(const lanelet::BasicPoint3d& end_point);
+ 
+  /*! \brief helper for traffic light Id
+   */
```

Comment:
Can you add a description of what this function is doing? And the param and return descriptions?